apps/meteor/app/lib/server/methods/saveSettings.ts

```diff
@@ -48,6 +48,16 @@ Meteor.methods<ServerMethods>({
 			const editPrivilegedSetting = await hasPermissionAsync(uid, 'edit-privileged-setting');
 			const manageSelectedSettings = await hasPermissionAsync(uid, 'manage-selected-settings');
 
+			// if the id contains Organization_Name then change the Site_Name
+			const orgName = params.find(({ _id }) => _id === 'Organization_Name');
```

Comment:
Should check if the Site_Name contains the default name, it could have been changed via env setting overwrite